DHT.h

```diff
@@ -41,11 +41,11 @@
 #endif
 
 /* Define types of sensors. */
-#define DHT11 11  /**< DHT TYPE 11 */
-#define DHT12 12  /**< DHY TYPE 12 */
-#define DHT22 22  /**< DHT TYPE 22 */
-#define DHT21 21  /**< DHT TYPE 21 */
-#define AM2301 21 /**< AM2301 */
+static const uint8_t DHT11{11};  /**< DHT TYPE 11 */
```

Comment:
`{11}` or `= 11` for compatibility?

Reply:
whats the difference? its still initialized the same?